src/components/Content/Counter/Ingredient/Ingredient.js

```diff
@@ -0,0 +1,74 @@
+import './Ingredient.css';
+import React from "react";
+
+class Ingredient extends React.Component {
+
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            ingredient: 0,
+            disabledMinus: true,
+            disabledPlus: false
+        }
+    }
+
+    addOne () {
+        this.setState((oldState) => {
+            if (this.state.ingredient === 4) {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient + 1,
+                    disabledPlus: !oldState.disabledPlus
+                }
+            } else {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient + 1,
+                    disabledMinus: false
+                }
+            }            
+        }) 
+    }
+
+    removeOne () {
+        this.setState((oldState) => {
+            if (this.state.ingredient === 5) {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient - 1,
+                    disabledPlus: !oldState.disabledPlus
+                }
+            } else if (this.state.ingredient === 1) {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient - 1,
+                    disabledMinus: !oldState.disabledMinus 
+                }
+            } else {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient - 1 
```

Comment:
And after that - do not use variables for simple conditions

You can refactor your code, so it is in the state you will have an array of ingredients. And push or remove elements from it whether it's + or -

After that - just .lenght property (and you do won't need to have separate value for that in state)